src/features/pastelDB/__tests__/pastelDBLib.test.ts

```diff
@@ -0,0 +1,488 @@
+import initSqlJs, { Database, QueryExecResult } from 'sql.js'
+
+import {
+  createBlock,
+  createBlocksubsidy,
+  createChaintips,
+  createListaddresses,
+  createListreceivedbyaddress,
+  createListtransactions,
+  createListunspent,
+  createMempoolinfo,
+  createMininginfo,
+  createNettotals,
+  createNetworkinfo,
+  createRawmempoolinfo,
+  createRawtransaction,
+  createStatisticinfo,
+  createTotalbalance,
+  createTransaction,
+  createTxoutsetinfo,
+  createWalletinfo,
+  insertBlockinfoQuery,
+  insertBlocksubsidyQuery,
+  insertChaintipsQuery,
+  insertListaddressesQuery,
+  insertListtransactionsQuery,
+  insertListunspentQuery,
+  insertMempoolinfoQuery,
+  insertMininginfoQuery,
+  insertNettotalsQuery,
+  insertNetworkinfoQuery,
+  insertRawmempoolinfoQuery,
+  insertRawtransactionQuery,
+  insertStatisticinfoQuery,
+  insertTotalbalanceQuery,
+  insertTransactionTableQuery,
+  insertTxoutsetinfoQuery,
+  insertWalletinfoQuery,
+} from '../constants'
+
+describe('managePastelDatabase', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('the database and tables should be created correctly', async () => {
+    const SQL = await initSqlJs()
+    const db: Database = new SQL.Database()
+
+    expect(db).not.toBeNull()
+
+    db.exec(createStatisticinfo)
+    db.exec(createNetworkinfo)
+    db.exec(createNettotals)
+    db.exec(createMempoolinfo)
+    db.exec(createRawmempoolinfo)
+    db.exec(createMininginfo)
+    db.exec(createBlock)
+    db.exec(createRawtransaction)
+    db.exec(createTransaction)
+    db.exec(createTxoutsetinfo)
+    db.exec(createChaintips)
+    db.exec(createBlocksubsidy)
+    db.exec(createWalletinfo)
+    db.exec(createListreceivedbyaddress)
+    db.exec(createListtransactions)
+    db.exec(createListunspent)
+    db.exec(createTotalbalance)
+    db.exec(createListaddresses)
```

Comment:
What is the purpose of this? If this is to initialize database structure, then it should be placed in `beforeAll`: https://jestjs.io/docs/setup-teardown

Reply:
Good comment, thank you.